### ISSUE TITLE
How to handle training data containing only a single type of interaction?

### ISSUE THREAD

**Issue by BubblyYi** [UX-Decoder/Segment-Everything-Everywhere-All-At-Once#69]
Thanks for the great work! 

I am very curious about how to handle situations where training data only partially intersects (e.g. none, textual, visual), the public dataset does not contain all the paired data required for current training. When encountering non-existent prompts(eg. gt training data is only image vision prompts, we do not have text, audio and reference image prompts),  does it use an attention mask to avoid query interacting with these non-existent prompts?

looking forward to your reply

**Comment by MaureenZOU** [comment]
I think we can handle that, however, you may need to edit the following code: https://github.com/UX-Decoder/Segment-Everything-Everywhere-All-At-Once/blob/v1.0/datasets/dataset_mappers/coco_panoptic_interactive_dataset_mapper.py
